src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -267,23 +278,39 @@ function DeploymentConfigForm({ respondOnSuccess, isUnSet }) {
                         </>
                     }
                 </div>
-
                 <div className="form__row form__row--code-editor-container">
                     <CodeEditor
                         value={tempFormData}
                         onChange={resp => { setTempFormData(resp) }}
-                        mode="yaml"
+                        mode={MODES.YAML}
+                        validatorSchema={schemas}
                         loading={chartConfigLoading}>
+                        <div className='readme-container'>
                         <CodeEditor.Header>
-                            <CodeEditor.LanguageChanger />
+                            <h5>{MODES.YAML.toUpperCase()}</h5>
                             <CodeEditor.ValidationError />
                         </CodeEditor.Header>
+                        {readme && <button className="readme-button pr-15 flexbox" type='button' onClick={e => setShowReadme(true)}>README<ArrowSquareOut className="icon-dim-18 scb-5 rotate " style={{ ['--rotateBy' as any]: '-90deg', marginLeft: '5px'  }}/></button>}
```

Comment:
Can we accommodate "{ ['--rotateBy' as any]: '-90deg', marginLeft: '5px'  }" into class rather than writing it in style directly?

Reply:
done